go.mod

```diff
@@ -1,4 +1,4 @@
-module gopkg.in/Netflix-Skunkworks/go-jira.v1
+module gopkg.in/jira.v1
```

Comment:
FYI, this is a breaking change; any existing v1 users will have their builds broken if they try to update to a newer v1 version. This is technically breaking semver, as you can only do such a breaking change as part of v2.

Of course, semver is subjective, and you can still merge this; I assume the number of module users are few. Though it would still be nice to do this massive rename as part of a v2, to be on the safe side.

Reply:
Okay, makes sense.  I think I will:
* release what is currently on `master`
* branch the current `master` to a `v1` branch,
* update this PR to use `gopkg.in/jira.v2` and tag master with `v2`.

I won't make a v2 release yet, we can decide if there are any other breaking changes that we want to get in before making a v2 release.  This does mean if we want to get a new v1 release out we will have to merge into the v1 branch and then probably rebase onto master.

Does that sound reasonable?